src/store/Permissions.ts

```diff
@@ -0,0 +1,246 @@
+import { Permission } from "../models/permission";
+
+const permissions: Permission[] = [
+  {
+    id: "activeTab",
```

Comment:
I would suggest removing `storage` and `identity` because they are not mentioned in the Chrome or Firefox UI nor actually give any meaningful access.

`activeTab` is also already mentioned in the Chrome UI as part of the extension dropdown menu.

If you would rather keep the non-revocable permissions in this menu, I think it would be useful to add a message saying "Can not revoke permission" instead of leaving it blank.

Reply:
Add an option to show required permissions

![image](https://user-images.githubusercontent.com/1621293/147909565-c58654b9-34cd-412d-b497-f5c1ef041690.png)

There's also a message to say the required permission cannot be revoked

![image](https://user-images.githubusercontent.com/1621293/147909604-618db92b-5ece-4034-9664-aadc845105a9.png)